project/plugins.sbt

```diff
@@ -14,3 +14,4 @@ addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.11.0")
 addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.4.9")
 addSbtPlugin("org.portable-scala" % "sbt-scala-native-crossproject" % "1.2.0")
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "2.0.6")
+addSbtPlugin("com.armanbilge" % "sbt-scala-native-config-brew" % "0.1.2")
```

Comment:
He-he

Reply:
Yeah Arman, stop doing all the work yourself 😛